File: SchoolAppClient.NG/src/app/Models/create-exam-schedule-standard-vm.ts
import { CreateExamSubjectWithScheduleVM } from "./create-exam-subject-with-schedule-vm";

export class CreateExamScheduleStandardVM {
  examScheduleStandardName!: string;
  examScheduleId!: number;
  standardId!: number;
  examSubjects: CreateExamSubjectWithScheduleVM[] = [];
}

File: SchoolApiService/ViewModels/CreateExamScheduleStandardVM.cs
namespace SchoolApiService.ViewModels
{
	public class CreateExamScheduleStandardVM
	{
		public string? ExamScheduleStandardName {  get; set; }
		public int ExamScheduleId { get; set; }
		public int StandardId { get; set; }
		public IEnumerable<CreateExamSubjectWithScheduleVM>? ExamSubjects { get; set; }
	}
}
